docker/dev/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Engelsystem PHP FPM development image including Xdebug
+FROM php:7.3-fpm-alpine3.10 AS php_fpm
```

Comment:
This should imho really depend on the `docker/Dockerfile` container to prevent having different PHP versions or missing modules when adding them only to the "production" image

Reply:
Now using the same base image as for the prod setup